app.js

```diff
@@ -14,12 +14,26 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
-
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index.hbs');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+  .then((beersFromApi) => {res.render('beers.hbs', { beersFromApi });
+    console.log('Beers from the database: ', beersFromApi)
+  })
+  .catch(error => console.log(error));
+})
+
+app.get('/random-beers', (req, res) => {
+  punkAPI.getRandom()
+  .then((beersFromRandom) => {res.render('random-beers.hbs', { beersFromRandom });
+  })
+  .catch(error => console.log(error));
+})
+
 app.listen(3000, () => console.log('🏃‍ on port 3000'));
```

Comment:
This comment is directed at the deleted package.json: when cloning this project, I cannot run "npm start", nor are dependencies installed when I execute "npm i". Please include a package.json by running "npm init" and then npm installing the dependecies used, or simply reference the package.json in the LAB repo.